integration-tests/README.md

```diff
@@ -111,20 +111,35 @@ This directory of the repository contains three sub-directories:
   - [Electron](./environments/electron/README.md)
 - [./realm-apps](./realm-apps): A directory of importable app configurations made available to the tests.
 
-### How to write tests
+### Tests needs to be explicitly imported
 
 Because of limitations (see below), we need to explicitly require in the files defining the tests: To write a new test, simply add it in the relevant file in `test/src/tests/` or create a new file and make sure to require that from `test/src/index.ts`.
 
-Tests will have access to the following globals:
+### Globals available
+
+Because we want out tests to run in a lot of environments, we've limited runtime available. Tests have access to the following globals:
```

Comment:
```suggestion
Because we want our tests to run in many environments, we've a limited runtime available. Tests have access to the following globals:
```